python/test/t_LinearModelAnalysis_std.py

```diff
@@ -52,6 +53,17 @@
         X[i, 0] * X[i, 0] + 0.1 * ot.DistFunc.rNormal()
 test = ot.LinearModelAlgorithm(X, Y)
 result = test.getResult()
+
+# Test various attributes
+cook = result.getCookDistances()
+cook_reference = [0.0233296,0.0360369,0.00178903,0.0502183,0.0966701,0.00562596]
+ott.assert_almost_equal(cook[0:6], cook_reference, 1e-5, 0.0)
+
+leverages = result.getLeverages()
+leverages_reference = [0.0864939,0.0797831,0.0735447,0.0677578,0.0624023,0.0574582]
+ott.assert_almost_equal(leverages[0:6], leverages_reference, 1e-6, 0.0)
+
```

Comment:
I think the right file to test these would be t_LinearModelAlgorithm not t_LinearModelAnalysis

Reply:
Test moved to t_LinearModelAlgorithm